### PR TITLE
V0 suggestor

### PR DESCRIPTION
**Pull request by Shruti-codes**

### CODE REVIEW

File: suggest.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import pandas as pd
+
+df = pd.read_csv("Pricing Calculator (2).csv")
```
Comment (sachin-netbook) [inline suggest.py:6]: Can you create functions for all the code?

def cheapest predictor( input_excel, input_instance):

File: suggest.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+print("\n \n ----------- Alternate cheapest instance on the same provider same region -------- \n")
+mp = df2[df2['Provider'] == prov]
+mp[mp[reg] == mp[reg].min()][cols]
+print(mp[mp[reg] == mp[reg].min()][cols].to_string(index = False))
+
+
+print("\n \n ----------- Alternate cheapest instance on the same provider any region -------- \n")
+cols1 = ['Price_US_East1', 'Price_US_West2', 'Price_US_East2', 'Price_AP-South1']
```
Comment (sachin-netbook) [inline suggest.py:34]: Move all these hardcodings to parameters or enums at the top of function.

File: suggest.py (hunk 1 above)
Comment (Shruti-codes) [reply suggest.py]: added

File: suggest.py (hunk 2 above)
Comment (Shruti-codes) [reply suggest.py]: done

**Review by sachin-netbook — APPROVED** [review]

**Pull request merged**